src/client/Microsoft.Identity.Client/Internal/Requests/DeviceCodeRequest.cs

```diff
@@ -51,7 +51,9 @@ internal override async Task<AuthenticationResult> ExecuteAsync(CancellationToke
             var response = await client.ExecuteRequestAsync<DeviceCodeResponse>(
                                builder.Uri,
                                HttpMethod.Post,
-                               AuthenticationRequestParameters.RequestContext).ConfigureAwait(false);
+                               AuthenticationRequestParameters.RequestContext, 
+                               // Normally AAD responds with an error HTTP code, but /devicecode endpoint sends errors on 200OK
```

Comment:
how weird!

Reply:
Yes. I thought of discussing this with ESTS folks, but I doubt they will change the behaviour of a production endpoint.